core/camel-componentdsl/src/generated/java/org/apache/camel/builder/component/dsl/SalesforceComponentBuilderFactory.java

```diff
@@ -1267,6 +1267,23 @@ default SalesforceComponentBuilder instanceUrl(
             doSetProperty("instanceUrl", instanceUrl);
             return this;
         }
+        /**
+         * Value to use for the Audience claim (aud) when using OAuth JWT flow.
+         * If not set, the login URL will be used, which is appropriate in most
+         * cases.
+         * 
+         * The option is a: &lt;code&gt;java.lang.String&lt;/code&gt; type.
+         * 
+         * Group: security
+         * 
+         * @param jwtAudience the value to set
+         * @return the dsl builder
+         */
+        default SalesforceComponentBuilder jwtAudience(
+                java.lang.String jwtAudience) {
+            doSetProperty("jwtAudience", jwtAudience);
+            return this;
+        }
```

Comment:
It looks like some code regarding the worker pool is missing, get it from https://github.com/apache/camel/commit/9ef5654a1ed6a5ef72cb67c19fdb0b468b21e1f2#diff-dd5f4218fc0e6e69abc63722905c8cceaafd31a80a6bca9b47efde0b7b3fac2f

Reply:
Okay, merged the commit above into this PR.